mudlet-mapper.xml

```diff
@@ -6646,6 +6646,21 @@ function mmp.getnums(roomname, exact)
     end
   end
 
+  local from = from or mmp.currentroom
+  if range and from then
+    if range == &quot;a&quot; or range == &quot;area&quot; then
+      local check = getRoomArea(from)
+      for i = #result, 1, -1 do
+        if getRoomArea(result[i]) ~= check then table.remove(result, i) end
+      end
+    elseif range == &quot;c&quot; or range == &quot;continent&quot; then
+      local check = table.concat(mmp.getareacontinents(getRoomArea(from)))
+      for i = #result, 1, -1 do
+        if table.concat(mmp.getareacontinents(getRoomArea(result[i]))) ~= check then table.remove(result, i) end
```

Comment:
This will only work if the rooms are on the same _combination_ of continents and the combination has _the same order_ as well. I'm not sure what determines the order here.